### PR TITLE
Thomas/rconv doc

### PR DESCRIPTION
**Pull request by Thomas-Ulrich**
update rconv readme (and another readme.txt I just noticed).

### CODE REVIEW

File: preprocessing/README.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 Preprocessing
 =============
 
-This folder contains all (1) tools required to set a up the input and generated source files required by SeisSol.
+This folder contains all<sup>1</sup> tools required to set a up the input and generated source files required by SeisSol.
```
Comment (uphoffc) [inline preprocessing/README.md]: reStructuredText has built-in support for foot-notes:
http://docutils.sourceforge.net/docs/user/rst/quickref.html#footnotes

Moreover, the footnote is a contradiction... wouldn't it make more sense to replace "all" by "most" and list in the footnote what is missing?

File: preprocessing/science/rconv/README.md [hunk 2]
```diff
@@ -2,7 +2,30 @@
 Rconv is a small tool which transforms file given in the Standard Rupture Format (SRF) to the intermediate NetCDF Rupture Format (NRF) which is required by SeisSol for simulating kinematic rupture models.
 
 ## Building rconv
-You need to have the proj.4 and the NetCDF libraries installed.
+
+rconv depends on the PROJ and NetCDF libraries.
+
+### Installing PROJ
```
Comment (uphoffc) [inline preprocessing/science/rconv/README.md]: Why don't we just state that rconv depends on version 4? Then a user can use an existing installation.

File: preprocessing/science/rconv/README.md [hunk 3]
```diff
@@ -2,7 +2,30 @@
...
+
+rconv relies on a deprecated version of the PROJ library (as it includes the file 'projects.h', which is not anymore built in recent releases of PROJ, see [OSGeo/PROJ#835](OSGeo/PROJ#835)). Also, the most recent version of PROJ need access to the internet when running cmake (because of the googletest 1.8.0 framework). To install PROJ on supermuc (behind a firewall), we therefore checkout an deprecated version of the PROJ library:
+
+```
+module load gcc
+git clone git@github.com:OSGeo/PROJ
+git checkout 3e8a1984e9ee662e74254704275c8e30791a5af0^1
```
Comment (uphoffc) [inline preprocessing/science/rconv/README.md]: I'd rather use a tag here, e.g. git checkout 4.9.3
(I checked that rconv compiles with this version)

File: preprocessing/science/rconv/README.md [hunk 4]
```diff
@@ -2,7 +2,30 @@
...
+mkdir build
+cd build
+cmake .. -DCMAKE_INSTALL_PREFIX=pwd
```
Comment (uphoffc) [inline preprocessing/science/rconv/README.md]: Should be \`pwd\`, right? Anyhow I'd suggest a proper installation directory here.

File: preprocessing/README.md (hunk 1 above)
Comment (uphoffc) [reply preprocessing/README.md]: Forget about the reStructuredText comment... just noticed this is a markdown file.

File: preprocessing/science/rconv/README.md (hunk 4 above)
Comment (krenzland) [reply preprocessing/science/rconv/README.md]: No, the standard way is `$(pwd)`

**Pull request merged**